sycl/include/CL/sycl/half_type.hpp

```diff
@@ -40,7 +40,6 @@ namespace detail {
 // bitselect builtin uses union with half specialization, but half has no
 // trivial constructor. This struct allow to use host half class instead of main
 // half class in the bitselect builtin.
```

Comment:
I guess this comment should be removed as well

Reply:
Removed